### DISCUSSION TITLE
Where can I find htaccess file

### DISCUSSION THREAD

**Discussion by ghost** [Kovah/LinkAce#514] [Support]
Hello, 

I followed [setup-without-docker](https://www.linkace.org/docs/v1/setup/setup-without-docker/)

And then the tuto said : 
> LinkAce already ships with a proper .htaccess file

However I do not find the file in the zip release. Do have any idea where it is ?

**Comment by Kovah** [comment]
Hi, the htaccess file is located in the `public` folder. However, if you run LinkAce in Docker behind an Apache web server, please take a look at https://www.linkace.org/docs/v1/setup/setup-with-docker/advanced-configuration/#running-linkace-behind-a-proxy-load-balancer

**Comment by juliend2** [comment]
@Kovah Well no, it's not that one.
In fact, you cannot tell Apache to change the path where a domain is hosted, only via a `.htaccess`. 

@ghost you're right, and that's an error in the docs.
There are two options:
1. (cleanest) go in your webhost and change the directory where the domain points
2. Use a redirect to go from domain.com to domain.com/public. Here's how: https://stackoverflow.com/questions/5792153/changing-the-root-folder-via-htaccess
